src/storage/src/hummock/compactor/fast_compactor_runner.rs

```diff
@@ -492,6 +492,7 @@ impl CompactorRunner {
             rest_data.next_sstable().await?;
         }
         let mut total_read_bytes = 0;
+        // TODO: use sst_size instead of file_size
```

Comment:
Will fast compact read the whole object or just the sst?

Reply:
For sst we can roughly assume that we need to read the whole Object (file_size), here it is just for logging, no need to be precise, I will remove the TODO